.github/workflows/markdown-link-check.yaml

```diff
@@ -0,0 +1,32 @@
+---
+"on":
+  push:
+    branches:
+      - trunk
+    paths:
+      - .github/markdown-link-check.json
+      - .github/workflows/markdown-link-check.yaml
+      - "**/*.md"
+  pull_request:
+    branches:
+      - trunk
+    paths:
+      - .github/markdown-link-check.json
+      - .github/workflows/markdown-link-check.yaml
+      - "**/*.md"
+  schedule:
+    - cron: "0 0 * * TUE"
```

Comment:
the cron is necessary since sometimes code refactors which don't otherwise touch markdown can break links.

See the refactor from `artichoke_backend::fs` to `artichoke_backend::load_path`.